internal/hud/renderer.go

```diff
@@ -239,38 +239,15 @@ func abbreviateLog(s string) []string {
 
 func (r *Renderer) renderResource(res view.Resource, rv view.ResourceViewState, selected bool) rty.Component {
 	layout := rty.NewConcatLayout(rty.DirVert)
+	renderResourceSummary(selected, rv, res, layout)
+	renderResourcesK8s(res, layout, rv)
+	renderResourceBuild(res, r, rv, layout)
+	return layout
+}
 
-	sb := rty.NewStringBuilder()
-	p := "  "
-	if selected {
-		p = "▼ "
-	}
-	if selected && rv.IsCollapsed {
-		p = "▶ "
-	}
-	sb.Text(p)
-
-	sb.Text(res.Name)
-	const dashSize = 35
-	sb.Fg(cLightText).Textf(" %s ", strings.Repeat("┄", dashSize-len(res.Name))).Fg(tcell.ColorDefault)
-	if res.LastDeployTime.Equal(time.Time{}) {
-		sb.Text("not deployed yet")
-	} else {
-		sb.Textf("deployed %s ago", formatDuration(time.Since(res.LastDeployTime)))
-	}
-
-	layout.Add(sb.Build())
-
-	if len(res.DirectoriesWatched) > 0 {
-		var dirs []string
-		for _, s := range res.DirectoriesWatched {
-			dirs = append(dirs, fmt.Sprintf("%s/", s))
-		}
-		sb := rty.NewStringBuilder()
-		sb.Fg(cLightText).Textf("  (Watching %s)", strings.Join(dirs, " ")).Fg(tcell.ColorDefault)
-		layout.Add(sb.Build())
-	}
+func renderResourceBuild(res view.Resource, r *Renderer, rv view.ResourceViewState, layout *rty.ConcatLayout) {
```

Comment:
maybe make `r *Renderer` a receiver rather than an argument, for consistency with the other funcs in this file
(or instead of either, make this method take a clock or a spinner so that it doesn't need `r`)